go.mod

```diff
@@ -8,21 +8,17 @@ go 1.13
 
 require (
 	github.com/cenkalti/backoff v2.2.1+incompatible
-	github.com/flimzy/diff v0.1.7 // indirect
-	github.com/flimzy/testy v0.1.17 // indirect
-	github.com/go-kivik/couchdb v2.0.0+incompatible
-	github.com/go-kivik/kivik v2.0.0+incompatible
-	github.com/go-kivik/kiviktest v2.0.0+incompatible // indirect
+	github.com/go-kivik/couchdb/v3 v3.2.5
+	github.com/go-kivik/kivik/v3 v3.2.3
```

Comment:
Had to update the `kivik` version. The old one had a bug that prevented bulk document retrieval from working.